File: unit-3/sprint-1/day-6_View Pager/assignments/ViewPager with circle indicator/app/src/main/java/com/example/imageslider/MainActivity.java
package com.example.imageslider;

import androidx.appcompat.app.AppCompatActivity;
import androidx.viewpager.widget.ViewPager;

import android.os.Bundle;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {
    ViewPager viewPager;
    ArrayList<Integer> imagesList = new ArrayList<>();
    ViewPagerAdapter viewPagerAdapter;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        viewPager = findViewById(R.id.viewPager);
        imagesList.add(R.drawable.img6);
        imagesList.add(R.drawable.img2);
        imagesList.add(R.drawable.img3);
        imagesList.add(R.drawable.img5);
        imagesList.add(R.drawable.img7);
        viewPagerAdapter = new ViewPagerAdapter(this,imagesList);
        viewPager.setPadding(100,0,100,0);
        viewPager.setAdapter(viewPagerAdapter);
    }
}

File: unit-5/Dagger-2/Dagger Example/app/src/main/java/com/example/dagger/MainActivity.kt
package com.example.dagger

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import dagger.internal.DaggerGenerated
import javax.inject.Inject

class MainActivity : AppCompatActivity() {

    @Inject
    lateinit var bike: Bike

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        
        bike.drivebike()
    }
}